dolphinscheduler-api/pom.xml

```diff
@@ -61,6 +61,11 @@
             <artifactId>dolphinscheduler-meter</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.dolphinscheduler</groupId>
+            <artifactId>dolphinscheduler-alert-server</artifactId>
+        </dependency>
+
```

Comment:
It's not a good idea to import alert-server in api-server, you should exposed a rpc method `testSend`  in alert server, and api call this method to test.

Reply:
Done